lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -369,14 +369,13 @@ def add_creditcard(post, creditcard, options)
             card[:cvc] = creditcard.verification_value if creditcard.verification_value?
             card[:name] = creditcard.name if creditcard.name
           end
-          post[:card] = card
 
-          if creditcard.is_a?(NetworkTokenizationCreditCard) && creditcard.source == :apple_pay
-            post[:three_d_secure] = {
-              apple_pay:  true,
```

Comment:
They do, but it's handled in https://github.com/activemerchant/active_merchant/pull/2146/files#diff-8299be227de5ba32c34c30c42a828c9cR376